pkg/core/resources/apis/meshservice/api/v1alpha1/deprecated.go

```diff
@@ -0,0 +1,21 @@
+package v1alpha1
```

Comment:
Isn't meshmultizoneservice and meshexternalservice supposed to have this aswell?